README.md

```diff
@@ -1,23 +1,27 @@
 # Calico Networking for rkt
 
 ## Building the plugin
-To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
+
+To build the Calico Networking Plugin for Rkt, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
 
 ## Using rkt Plugins
+
 The [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins in rkt will walk you through the basics of setting up networking in rkt.
```

Comment:
Rather than having this header, this could just be a part of the brief description I mentioned above, pointing at the rkt docs for more information.